README.md

```diff
@@ -14,88 +26,150 @@ To get started with the Ada Trader project follow these steps:
-
-1. The `QuoteView` class should:
-  * Have an `events` property that defines two handlers for the `click` event, one for each `button` in the quote template.
-  * Have a function that runs when the `click` event happens on the buy button. This function should:
-    * Increase the stock's price by $1.00.
-    * Re-render the view so that the new price is displayed to the user.
-  * Have a function that runs when the `click` event happens on the sell button. This function should:
-    * Decrease the stocks' price by $1.00.
-    * Re-render the view so that the new price is displayed to the user.
-1. The `Quote` model should have unit tests written in Jasmine for any custom methods you add to it. -->
-
-<!-- ### Wave 4 - Events: Simulate
-This wave will implement a simulation of the broader stock market. To make this simulation as true-to-life as possible, we'll be randomly moving the stock price up or down by a small amount, once per second. In `app.js`, we've provided the function `simulate` which does exactly that. In order to achieve this, your application should be updated so that:
-
-1. The `QuoteView` class should:
-  * Setup [an event listener](http://backbonejs.org/#Events-on) for a custom event named `change:price`. This should be done in the `initialize` function.
-  * Have a function that runs when the `change:price` event happens. This function should:
-    * Expect that a `change` parameter will be passed into it.
-    * Expect that the `change` parameter will be a number, positive OR negative.
-    * Add the `change` parameter onto the current stock price.
-    * Re-render the view so that the new price is displayed to the user.
-1. Something in the project should call the `simulate` function on each of the `QuoteView` instances.
-
-You may find that you would rather move the `simulate` function out of `app.js`. Feel free to do so.
-
-We do not expect you to write unit tests for the `simulate` function. -->
+### Project Baseline
+This project uses the same structure as BackTREK and is based on our [backbone baseline](https://github.com/AdaGold/backbone-baseline). The baseline for this project includes some model and collections that have already been implemented:
+
+| Model | File | Purpose |
+|:------|:-----|:--------|
+| `Quote` | `src/models/quote.js` | This model represents an individual stock quote, and has attributes for:<br>• `symbol` - code name for the stock (e.g. AAPL)<br>• `price` - current price of the quote<br>You will add the ability to buy / sell a quote using custom functions. |
+| `QuoteList` | `src/collections/quote_list.js` | This collection represents all of the quotes available on the trading platform. |
+| `Simulator` | `src/models/simulator.js` | This model handles the logic for simulating stock market activity by randomly shifting the price for each quote up or down each second. You will need to connect it to your `QuoteList` instance in `src/app.js`. |
+
```

Comment:
There's a lot of text in this document, maybe some pictures or diagrams would help.  I'm thinking maybe a diagram of the objects in the structure and their connections.  

I can see a lot of students not knowing how to start connecting things with events.  Maybe that's the level 2 part, but it will be a pain point.